README.md

```diff
@@ -7303,6 +7303,30 @@ try clearDBQueue.inDatabase { db in
 // Now the copy is done, and the clear-text database can be deleted.
 ```
 
+**Advanced configuration options for SQLCipher**
+There are two advanced configuration options that you can set for configuring SQLCipher that control aspects of the encryption and key generation process.
+
+```swift
+var configuartion = Configuration()
+configuration.passphrase = "secret"
+configuration.cipherPageSize = .pageSize4K
+configuration.KDFIterations = 128000
+let dbQueue = try DatabaseQueue(path: "...", configuration: configuration)
+```
+***cipherPageSize***
+The `cipherPageSize` is used to adjust the page size for the encrypted database (this corresponds to the SQLCipher `PRAGMA cipher_page_size` configuration option). Increasing the page size can noticeably improve performance for certain queries that access large numbers of pages. 
```

Comment:
Is it possible to add a link to the SQLCipher documentation (here and for kdfIterations below)?